plugins-structure/README.md

```diff
@@ -0,0 +1,579 @@
+# PoliteiaGUI - Plugins Structure
+
+A Guest User Interface configurable structure for creating multiple applications
+using Politeia's [[1]](#1) plugin architecture.
+
+## Index
+
+0. [Requirements](#requirements)
+1. [Introduction](#introduction)
+2. [GUI Composable Plugin Architecture](#gui-composable-plugin-architecture)
+3. [Application Shell](#application-shell)
+4. [Core](#core)
+
+## Requirements
+
+The following guide will assume you are familiar with the tools listed below. We
+recommend you to take some time to study them before proceeding.
+
+- [JavaScript](https://developer.mozilla.org/docs/Web/JavaScript)
+- [Micro-Frontends](https://micro-frontends.org/)
+- [Lerna](https://lerna.js.org/)
+- [Yarn Workspaces](https://yarnpkg.com/features/workspaces)
+- [Politeia](https://github.com/decred/politeia)
+- [Redux Toolkit](https://redux-toolkit.js.org/)
+- [Window History](https://developer.mozilla.org/docs/Web/API/Window/history)
+
+## Introduction
+
+Politeia is the underlying platform that provides a versioned, timestamped data
+store [[2]](#2), which can be used for creating new applications on top of it.
+
+> The original motivations behind Politeia were to create a platform that would
+> allow the Decred stakeholders to manage the Decred treasury. This platform
+> would let individuals and teams submit proposals (i.e. requests for funding
+> from the treasury) and would facilitate the Decred stakeholders voting on
+> whether to fund the proposal or reject it. The data would be stored off-chain,
+> but would be episodically timestamped onto the Decred blockchain" - _Politeia:
+> Pi 2021 Q3 (Proposal System)_ [[2]](#2).
+
+However, the stored data could be any set of data, not exclusively proposals,
+which could also be used as a more generic storage, such as contractors
+payments management, proposals discussions, etc.
+
+The current design would allow developers to create new plugins to interact,
+extend and manage the Record functionality. Therefore, we can use Politeia
+[[1]](#1) architecture to create more applications, such as
+[Decred's Proposal system (Pi)](https://proposals.decred.org) and
+[Decred's Contractor Management System (CMS)](https://cms.decred.org/)
+_(CMS not implemented on the new architecture yet)_.
```

Comment:
I think these paragraphs doesn't add much to the text itself. I would rephrase them to something like:

`A lot of applications can benefit from this design, that's why we decided to create a composable plugin architecture. Developers are now able to create plugins to interact, extend and manage the records stored and create awesome applications with it.`